beets/library.py

```diff
@@ -1482,10 +1481,91 @@ def tmpl_upper(s):
         """Covert a string to upper case."""
         return s.upper()
 
+    # Regular expressions used by tmpl_title
+    SMALL = 'a|an|and|as|at|but|by|en|for|if|in|of|on|or|the|to|v\\.?|via|vs\\.?'  # noqa: E501
```

Comment:
Instead of `#noqa`ing this line, it could just be broken up onto two lines:

```py
SMALL = 'some stuff here' \
  'more stuff here'
```

Reply:
Hi @sampsy. That's correct, it was taken from that repository and has been modified. I considered adding a dependency, however I found that that it is not as well maintained and has some unnessesary code. Do you think it would be worth adding in this custom version for beets?